tools/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
Cleaning up **.gitignore** files should be done through an ad-hoc ticket/PRs.

Reply:
We should only have one `.gitignore` per repo, in the top level project.